src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,325 @@
 /**
 This protocol defines common types used in the other GA4GH protocols. It does
 not have any methods; it is merely a library of types.
+
```

Comment:
@diekhans Can you do a PR against CONTRIBUTING.md for that?

I think we don't indent for the protocol because then the doc comments on the objects in the protocol would be indented. This was found to confuse Avrodoc, because it doesn't properly strip the leading spaces on the comment lines, and we then get something silly like the entire docstring in a markdown code block when we generate the API docs. Maybe if we switch over to Pandoc or something else for doing the API docs, we can change this.